cicd-scripts/helpers/run_without_ui.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+SPIDER_PYTHON_VERSION=3.12
+sudo bash -c "nohup /usr/local/bin/python${SPIDER_PYTHON_VERSION} ./search_gov_crawler/scrapy_scheduler.py > /var/log/scrapy.log 2>&1 &"
```

Comment:
does this work? usually you would want to point to the python in the venv?

Reply:
Good call!

I fixed this by activating the virtual environment